x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/import_prebuilt_rule.cy.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  expectToContainModifiedBadge,
+  expectToNotContainModifiedBadge,
+  importRulesWithOverwriteAll,
+} from '../../../../tasks/alerts_detection_rules';
+import { TOASTER } from '../../../../screens/alerts_detection_rules';
+import { createRuleAssetSavedObject } from '../../../../helpers/rules';
+import { deleteAlertsAndRules } from '../../../../tasks/api_calls/common';
+import { createAndInstallMockedPrebuiltRules } from '../../../../tasks/api_calls/prebuilt_rules';
+
+import { login } from '../../../../tasks/login';
+
+import { visitRulesManagementTable } from '../../../../tasks/rules_management';
+
+const RULES_TO_IMPORT_FILENAME = 'cypress/fixtures/8_18_prebuilt_rules.ndjson';
+
+describe(
+  'Detection rules, Prebuilt Rules Import workflow - With Rule Customization',
+  {
+    tags: ['@ess', '@serverless', '@skipInServerlessMKI'],
+    env: {
+      ftrConfig: {
+        kbnServerArgs: [
+          `--xpack.securitySolution.enableExperimental=${JSON.stringify([
+            'prebuiltRulesCustomizationEnabled',
+          ])}`,
+        ],
+      },
+    },
+  },
+  () => {
+    describe('when file is unmodified prebuilt rule with matching rule_id', () => {
```

Comment:
I think we could combine both tests in this file into a single test, where we'd test importing a combination of different rules:

- prebuilt non-customized with a base version available, but without the rule being installed
- prebuilt customized with a base version available, but without the rule being installed
- plus the same, but with the rule being already installed
- plus the same, but with a missing base version
- custom

It could be a small number of cohesive tests too, but the point is to test importing multiple different rules at once - that's how we make these tests faster, lighter, and more high-level.

We will have single-purpose scenarios automated at the API integration level.